### PR TITLE
Crisis in Management, an alternate start.

### PR DESCRIPTION
**Pull request by leklachu**
**Content (Missions / Human Storyline)**

## Summary
With some encouragement from @samrocketman (thanks!) I'm offering my Syndicate-based storyline for inclusion in to the main game. It's a complete path through the civil war, starting from an alternate start within the Syndicate.

The campaign is complete...ish. There's a few things that need tidying up for inclusion into the main game, and I'm happy to do that if there's in interest to include this. This is a draft PR to gauge that interest.

The plugin, including various notes, is at https://github.com/leklachu/endless-sky-syndicate

## Style

From the outset this was a storyline about the character development of the player-character, though I didn't pull that off as well as I hoped. It's also more wordy than the original FW campaign (though the FW rewrite is pushing more verbose), and much less combat-orientated. Also,

- Spellings are British. I figured this was okay for a plugin. Then again, you Americans like Brits as your baddies in movies, so maybe it's appropriate the Syndicate talk in British spellings!
- Several missions depend on the player's identity from the alternate start.

## Alternate Start or Main Campaign

As far as I am aware there are two other attempts for a main Syndicate campaign: @Amazinite  has been planning one at https://github.com/Amazinite/ES-Syndicate-Campaign and just now I saw @IsocratesGrey's proposal in #7864. Neither have got beyond an outline and a beginning, but both, it seems, would prefer their own over mine. I have no complaint with that! And would welcome their efforts becoming the main Syndicate campaign.

On the one hand, from @IsocratesGrey, this storyline "doesn't seem to fit the same plot beats and so feels out of sync with the FW campaign." On the other, from @samrocketman, "your work does a great job providing syndicate story in a way that melds nice with the core story."

So to open discussion:

There are two ways this could be included. Either much as it is, as an alternate start; or altered to be accessible to the usual pilot from New Boston. It could even do both, having an altered version accessible to the normal pilot, and a special version (keeping all the current content) from an alternate start.

## Compatibility with the main game

There are a few points of dubious compatibility, mostly because parts of the game assume the player has played through one or other of the Free Worlds branches. Some of this ought to be fixed in main game anyway, to make it easier for future Navy and Syndicate campaigns which anyway are intended to come in due course.

There may also be parts of my campaign some feel should be changed to fit the main canon. (Though to my knowledge there's nothing that directly breaks canon, nor sits far outside of the ES canon as it stands.)

If this is included just as an alternate start, and other Syndicate campaign(s) are written, it might be good to tie these together. Either I can include characters and events from the main Syndicate campaign once it's developed, or they can use characters from mine. I'd love to have more points of crossover where one story answers questions left open from another - e.g. like this story involves some of the Syndicate higher-ups alluded to in FW story, and the start even makes the player be an instrument in the terrorist attack framing.

## Conclusion

Basically, I want to know if you'd like this in the main game! If so, I'll get to work making it ready, and welcome anyone who wants to help.

### CODE REVIEW

File: data/human/cim-events.txt [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+		has "event: fw northern expansion"
+		has "ScS Revenge sweet Revenge: offered"
+	source
+		government "Free Worlds" "Republic" "Syndicate" "Independent" "Neutral" "Quarg"
+	on offer
+		log `The war grows more bloody, with the Free Worlds claiming several worlds to the north of their original territory.`
+		conversation
+			`When you land on <origin>, the latest news from the war is filling the headlines. It seems the Free Worlds have occupied more systems to the North of their original territory. Whether the defecting worlds invited this or the Free Worlds annexed them against their will, seems to be a matter of some dispute.`
+
+# syndicate are normally neutral to alphas.... not sure how much this is intentional, and how much because nobody's needed to consider it. For now, hostility is set and unset: but this is risky: if main-game changes this later, thi s will override!
```
Comment (Hurleveur) [inline data/human/cim-events.txt:59]:
```suggestion
# syndicate are normally neutral to alphas.... not sure how much this is intentional, and how much because nobody's needed to consider it. For now, hostility is set and unset: but this is risky: if main-game changes this later, this will override!
```
also I wouldn't do that, just make them hostile if they're going to be hostile in this campaign, if we ever want alpha to not be attacked by anyone we can just give them `marked`

File: data/human/cim-events.txt [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+	government Syndicate
+		"attitude toward"
+			"Alpha" -.01
+
+event "ScS syndicate neutral to alphas"
+	government Syndicate
+		"attitude toward"
+			"Alpha" 0
+
+event "ScS pug invasion 3"
```
Comment (Hurleveur) [inline data/human/cim-events.txt:71]: just wondering, why cant we re-use the FW event for this one?

File: data/human/cim-extras.txt [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+	"player reputation" -100
+	"attitude toward"
+		"Republic" -.01
+		"Syndicate" -.01
+		"Free Worlds" .01
+		"Militia" .01
+	"friendly hail" "friendly militia"
+	"hostile hail" "hostile militia"
+
+government "Sawyer Escape"
```
Comment (Hurleveur) [inline data/human/cim-extras.txt:17]: governments all need to be in government.txt

File: data/human/cim-outfits.txt [hunk 4]
```diff
@@ -0,0 +1,29 @@
+outfit "Blaze-Pike Turret (modified)"
```
Comment (Hurleveur) [inline data/human/cim-outfits.txt:1]: all human outfits are in outfits.txt, and for one outfit it's really not worth spliting (ik, was a plugin, no worries)

File: data/human/syndicate missions.txt [hunk 5]
```diff
@@ -0,0 +1,136 @@
+mission "Weapon Salesman"
```
Comment (Hurleveur) [inline data/human/syndicate missions.txt:1]: you forgot the copyright text we have in at the start of all files here, and everywhere else

**Review by Hurleveur — COMMENTED** [review]
just took a quick glance

**Comment by RisingLeaf** [comment]
Well that comes unexpected, but in my opinion it is better to have someone else doing this than Derpy having yet another big project.

But anyways, my two cents just from reading the description:
- Not too sure about that British spell, not because I don't like British but for consistency reasons. (Also this project is pretty international from what I see, so there is no shared hate against brits XD)

- Alternativ start? Sure, but separate that from the syndicate campaign. In my opinion every start should make every campaign possible, respective to the point in time the start is.

So yea I am not the guy for theme and all that stuff so this is a personal opinion.

**Comment by Terin** [comment]
> Not too sure about that British spell, not because I don't like British but for consistency reasons. (Also this project is pretty international from what I see, so there is no shared hate against brits XD)

And the CI _hates_ it. Also, the impression you get of the Syndicate executives through the Free World campaign feels pretty American, so changing it over would be for the best.

But yeah, the required alternate start, while providing a really great character for the player to roleplay and jump into that campaign, is probably a deal-breaker. We could have two Syndicate campaigns, but that's a lot of reviewing burden on the game. We could maybe tweak it so mill worker <first> <last> can get into this one, but from the bits I've skimmed over, it seems like there'll be a _big_ gulf to cross for that.

But I'll let others bring up their thoughts as well, especially Amazinite.

File: data/human/cim-events.txt (hunk 2 above)
Comment (leklachu) [reply data/human/cim-events.txt:71]: Probably that something in FW triggers off whether that event has happened, and without amending FW code (from my plugin!) I had to make a separate-named event. I think I did that a couple of times.

Comment (leklachu) [reply data/human/cim-events.txt:71]: I don't have time now, but all those niggles I'll have to go through and work out the best way to fix for vanilla inclusion. Thanks for bringing it up.

**Comment by leklachu** [comment]
> And the CI hates [British spellings].

Yeah, I noticed that once I submitted. I have no problem changing all of that to US spelling. I agree consistency is good.

About tweaking this for a main-start pilot: Most of it works to adjust simply. (It'll take time, because I'll need to run through the entire campaign text!) Stuff with Uncle WC, I can make a different reason to meet him, for a main-start pilot. Then all the side-plot of his wives is probably best to truncate or cut out, and enable if the player chooses the alternate start. The harder thing might be the pilot's progression from bigoted money-grubber to compassionate human - but as I say, I didn't write that half as well as I hoped, so maybe I don't mind cutting down whatever I can't make fit for a generic pilot. I guess I could have more dialogue options to let the PC choose their personality as they go along.

**Comment by Terin** [comment]
I do love the idea of <first> <last> being able to turn into a _real_ goblin via the Syndicate campaign, but the arc there would probably be starting off as a reasonable person before becoming an asshole if you go down a dark path, which doesn't sound like where yours ends up with.

That's great to hear that it could be tweaked for a normal pilot, though. We have support for unlockable starts now, with the big one in #7721 giving you a start that skips past the human and Hai campaigns. I like the idea of getting the start here as a reward after you complete a human campaign for the first time, giving you a lot of starting net worth that could be brought into the Syndicate for the style it has right now.

File: data/human/cim0-bootstraps.txt [hunk 6]
```diff
@@ -0,0 +1,738 @@
+start "syndicate"
```
Comment (RisingLeaf) [inline data/human/cim0-bootstraps.txt:1]: all starts go in the same file, I think

**Review by RisingLeaf — COMMENTED** [review]
two small comments

**Comment by leklachu** [comment]
> what is the lore behind the "mini-wormhole"?

Good point. I'd forgotten: this was something I deliberated on for some while, and came up with this solution that was good enough for a plugin but would need consideration for the main game.

I prefer the idea that even the Pug can't create wormholes on a whim, maybe taking even decades or more of planning and preparation. So this was a hack: a kind of blend between: maybe the Pug have some means for their own convenience (hidden wormholes? something else?) besides jump drives of getting in and out, and this either is that or piggy-backs off it; And/Or, this is a kind of small, unstable version of the normal wormhole we see, something the Pug can hold open for a single ship, but not sustainable as a general wormhole.

If this is to go into vanilla I agree it needs some consensus and probably minor adaptation. As a plot device in this situation I like it, it works better than the pug giving a jump drive, but I'm also willing to rewrite it out if necessary.

**Comment by leklachu** [comment]
So, now it's been a few days, any more thoughts on this? Any sort of consensus on whether this might be welcome,

a) not at all in the main game,
b) adjusted to fit as the main Syndicate campaign, plus an optional alternate-start for extras, or
c) as a unique storyline only from an alternate start.

@Amazinite especially, and @IsocratesGrey, what are your thoughts?

I am happy to do the adjustments to make it fit as a main campaign, and work in other changes necessary, if this is wanted; else I can just leave it as is as a plugin (and hopefully improve/fix from time to time!).

**Comment by IsocratesGrey** [comment]
Hi, thanks for tagging me in, I've been busy with work so haven't been keeping up on this enough.

I've read through it again now and I think that as it is it works best as an alternate start, but with adjustments it could merge into the syndicate campaign using @Amazinite  start, though that would need some reworking of the early part of it.

I still have concerns about the story beats for the first half of the campaign, it seems like it doesn't really line up with the pacing of the FW plot, but I love the Pug war portion, it feels stronger and really sits nicely inside what's happening in the FW campaign.

You seem to be lacking some throughlines in your story, things that connect the player to the story at multiple times, like how Katya and Eyes are used in the FW campaign. Reoccurring characters, or characters who start as minor then become major, like Danforth. There are some strong contenders, the trophy wives (tm) of the uncle could continue through it, they are key players in parts 1 and 2, but then they disappear. Mary-Jane has the same problem, being core to 2 and 3 then disappearing in 4. Meanwhile Arnold seems to come out of nowhere in 2 and becomes a main character, and instead of any of the characters we know from before the people we're trying to save in 4 are just two random twins who were kidnapped and don't really have a connection to the plot.

You may want to create a map of your characters, where they appear, and what their purpose in the story is, it will probably show you some characters that could be introduced earlier as minor characters only to have them become major later, or characters that can be merged, or brought back. With the number of wives the uncle has perhaps one of them is kidnapped by the alphas?

**Comment by leklachu** [comment]
Thanks for that write-up, that's helpful, and I see I'd have more to do to make it fit the main start. One of the fun things about making an alternate start was putting the player in a different story to the main one, and having you intersect with the main story from time to time. The galaxy is big, and this is a different story in the same timeline, though in fact it still has very much to do with the civil war. And there are occasional references to what the original player is doing in the background: such as when an 'eerily familiar' captain walks past with a FW leader, or, and I shall be sad to lose this in a mainlined adaptation, the interaction with Joe and Maria, who are only on Millrace because the original player rescued them from Thule(?) shortly before.

Kind of makes me prefer to keep it as an alternate start; but I am still happy to adjust to a mainline story. (And keep an alternate start for the missing bits.)

About merging with Amazinite's start, I'll need to have a look at that. IIRC it also includes its own alternate start. Anyhow that'll be good if it works as a way to link this to the main New Boston start. In any case I'd cut out my one's intro (up till your debt's paid off) to be purely alternate start.

As to the characters, you have a good point; but, there's certain aspects I like about some of the timing of my characters' introductions. I'll have to think on that. The Uncle's wives I don't see working for the main campaign, because they're written around WC being your uncle, which (unless we want to canonise First Last's relatives!) doesn't apply to the main player.

Maybe the answer is, if we'd like this for a main Syndicate campaign, to take about half of it and rewrite to a full campaign more in keeping with the FW campaign. ...Then again, if what you most like is the Pug war portion, that's really a separate-but-parallel story to the galaxy's main concerns. Unless you mean the chasing-down-extremists part before that.

Perhaps a big question is: how much do we want branch options for the main player to allow them to take different parts in the same story through the galaxy, and how much do we want options to let them take different stories through the same timeline.

**Review by samrocketman — COMMENTED** [review]
In favor of the alternate start,

The syndicate start has a 10m initial loan.  At first, one might balk, but I changed my play style interestingly.  The daily loan payments are always greater than your income.  You start min-maxing by purchasing alternate goods to burn up credits (move your wealth into assets like outfits or cargo).  Eventually, I built up enough non-credit assets to sell them off and buy second ship.  This process repeated until quickly growing a fleet and naturally paying off the 10m credit debt.

It has a tutorial which results in a 10m credit job but I had it paid off by the end of the tutorial with that strategy; keeping the star queen.  So it was just a 10m credit gift. The story intro is pretty compelling. This put me in that mindset through story which I think is amazing.  _To change player behavior through story_ away from how I normally play.

The starter ship is a downgraded star queen so it enables that style with the type of ship (terrible outfits and barely passable slow small engines).  It is the story of a failed entrepreneur.

### Summary

I think it would be a travesty to deny players that interesting start with the back-story.  Probably fine if you want to interweave the syndicate story with the main start but to cut it out completely I think would not be very fun.  The option to have the alternate start should definitely remain.

**Comment by leklachu** [comment]
So, it's been some while now, but I've been mulling this over in the background. The more I think about it, the more I think it's not the best way to go for the main Syndicate campaign. This is the story of a different pilot, while OP First Last is off-stage joining the Free Worlds and fighting the civil war. Here the civil war, and the epic alliance against the Pug, are in the background, as you go on your journey of profit, compassion and salvation. And the story progression reflects that (sometimes).

The official Syndicate campaign is the story of what if Cap'n F. Last  helps the Syndicate instead, and joins the civil war that way. Things turn out a bit differently. You're not the only fish in the pond, but you are a big one by the end - "hero of the republic" and all that. (Your reputation in the subsequent Hai Reveal story.) There are as many ways to write this as there are choices a person could make, but for the game, I'd say: the player joins the Free Worlds to fight for idealism, justice and freedom; or joins the Navy for peace through stability, and a trust in democracy; or the Syndicate...? for profit! Or an idealistic love of pure, unbridled capitalism. Of course, it needn't stay that way ;-)

One other point I'd make. I'm a bit concerned about the extended writing in Hai Reveal and the updated FW story. The wiki does (did?) say somewhere about writing a game not a novel, and I'm afraid we may be moving the wrong way with overdone story exposition and text-based ornamentation. This 'Crisis in Management' is also quite wordy, but some of the writing I'm really quite proud of, and would be sad to lose - but maybe its place isn't in a main campaign.

So, for a Syndicate campaign: I'm going to have a try, and perhaps I can write something worthy of inclusion. I'll look at Amazinite and IsocratesGrey have written, and of course my own work, but start fresh and include things only as/where they fit.

For this PR, then, I'd love to have Crisis in Management included as an alternate start, unlockable after you've completed the main campaign. The intro could be made a general alternate start, from which you can take any path (though the <mumble><mumble><spoilers><spoilers> would be quite amusing when later travelling with Katya!) but the rest would be best I think locked behind alternate start. I don't think there's a problem having "two Syndicate campaigns" because this isn't a second Syndicate campaign: it's a separate and additional story in the ES galaxy, as the various alien races are separate stories in the galaxy.

So if that's wanted, I can get to work tidying it up for inclusion. And if not, I'll let it stand as a plugin. If we can get a consensus "No, that's not wanted," then we can close this PR.

(I remain open to others' suggestions, of course!)

**Comment by Saugia** [comment]
There hasn't been any activity since May on this PR, and we think this would best remain as a plug-in as an alternate to what's planned for the Syndicate campaign.

**Comment by leklachu** [comment]
Thanks. I've been thinking the same.

By the way, has there been any further development on the Syndicate campaign? I keep having ideas I'd like to write for it, but I very much doubt I'll have time to put in proper effort to it for quite some time now.

**Comment by bene-dictator** [comment]
No further development yet, Derpy is looking at it after HR revision and FW revision pt 2 so some time away. I am also eyeing it but I'm gonna be out of action for a little (I hope I mean it this time) so the floor is yours. But share your ideas if you got some.

Derpy has a repo with his old work on the Syndicate campaign if you want some structure. But it's quite old.
